frontend/server/src/DAO/Contests.php

```diff
@@ -462,9 +463,9 @@ final public static function getContestsParticipating(
                 pi.problemset_id = p.problemset_id AND
                 pi.identity_id = ?
             LEFT JOIN
-                Problemset_Identities
+                Problemset_Identities contestants
```

Comment:
```suggestion
                Problemset_Identities AS contestants
```

faltan todos los `AS` para los aliases de las tablas. de paso puedes agregar los que faltaron en https://github.com/omegaup/omegaup/pull/5405#discussion_r633497995 ?

Reply:
Claro faltan las pruebas, creo que puse por error ese check cuando dupliqué el PR en base al de la columna de "organizador".

Y si, agrego el "AS".